chapters/scoping.tex

```diff
@@ -135,6 +135,19 @@ \subsection{Composite Name Lookup}\doublelabel{composite-name-lookup}
 \autoref{redeclaration} for further explanations regarding looking inside
 partial packages.
 \end{nonnormative}
+\begin{example}
+Components and classes are part of the same name-space and thus a component may
+not have the same name as its class or the first part of the class-name as that
```

Comment:
Just to clarify a tiny bit that this is an example showing a consequence, as opposed to an example containing a rule:
```suggestion
Components and classes are part of the same name-space and thus a component cannot
have the same name as its class or the first part of the class-name as that
```